butter_cms/content_field.py

```diff
@@ -7,8 +7,21 @@ def __init__(self, auth_token):
         Client.__init__(self, auth_token)
         self.path = 'content/'
 
-    def get(self, keys=None, params=None):
+    def get(self, keys, params=None):
```

Comment:
Why `keys` is now required?

Reply:
Because as you can see in tests, when no keys are provided, result is empty. This will at least guide developer that this should be used with some keys.

Reply:
> This will at least guide developer that this should be used with some keys.

Hmm, maybe we could indicate it better by using type annotations, WDYT?

Reply:
This is similar as indicating that `search` method on `author` or `post` requires `query`. It would be definitely better to use type annotations but IMO that would be outside the scope of this PR.

Reply:
Ok, makes sense.